model-uploader/src/main/java/ai/djl/uploader/MetadataBuilder.java

```diff
@@ -451,6 +453,16 @@ private Artifact constructArtifact(Path destination) throws IOException {
             item.setUri(uri);
             files.put(names[0], item);
         }
+        if (application == Application.CV.IMAGE_CLASSIFICATION
```

Comment:
Can you do this when you do gluoncv import. Even if they are using same dataset, they may not have the same synset.

Reply:
all gluon cv image classification with imagenet has the same synset.
I didn't add synset for cifar10

Reply:
> all gluon cv image classification with imagenet has the same synset.
> I didn't add synset for cifar10

But a keras model may have a different synet even if it is `imagenet`. synset is a platform specific class

Reply:
image classification usually requires "synset", we only add this if "synset" is missing